app/assets/javascripts/close-flash.js

```diff
@@ -0,0 +1,7 @@
+//TODO: not be lazy and do this properly
+
+$(document).ready(function() {
+    $(".close-flash").click(function() {
+        $(this).parent().css("display","none");
+    });
+});
```

Comment:
This is a great addition! Since jQuery is baked into Rails maybe we could use
```js
 $(this).parent().fadeOut();
```
for a nicer effect?